style/style.go

```diff
@@ -16,14 +16,52 @@ func Color(value string) vecty.Markup {
 	return vecty.Style("color", value)
 }
 
+func Width(size Size) vecty.Markup {
+	return vecty.Style("width", string(size))
+}
+
+func MinWidth(size Size) vecty.Markup {
+	return vecty.Style("min-width", string(size))
+}
+
+func MaxWidth(size Size) vecty.Markup {
+	return vecty.Style("max-width", string(size))
+}
+
 func Height(size Size) vecty.Markup {
 	return vecty.Style("height", string(size))
 }
 
+func MinHeight(size Size) vecty.Markup {
+	return vecty.Style("min-height", string(size))
+}
+
+func MaxHeight(size Size) vecty.Markup {
+	return vecty.Style("max-height", string(size))
+}
+
 func Margin(size Size) vecty.Markup {
 	return vecty.Style("margin", string(size))
 }
 
-func Width(size Size) vecty.Markup {
-	return vecty.Style("width", string(size))
+type OverflowOption string
+
+const (
+	OverflowVisible OverflowOption = "visible"
+	OverflowHidden                 = "hidden"
+	OverflowScroll                 = "scroll"
+	OverflowAuto                   = "auto"
```

Comment:
I believe your intent was to make all these `const`s have type `OverflowOption`.

However, since you're not using `iota`, only the first entry has type `OverflowOption`. All others are untyped strings. Fix it by doing:

```Go
const (
	OverflowVisible OverflowOption = "visible"
	OverflowHidden  OverflowOption = "hidden"
	OverflowScroll  OverflowOption = "scroll"
	OverflowAuto    OverflowOption = "auto"
)
```

An alternative form is:

```Go
const (
	OverflowVisible = OverflowOption("visible")
	OverflowHidden  = OverflowOption("hidden")
	OverflowScroll  = OverflowOption("scroll")
	OverflowAuto    = OverflowOption("auto")
)
```

But I don't think it's necessarily better. Just sharing for posterity.